ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -363,6 +363,19 @@ public void testSimpleBlobRawMode() throws Exception {
     }
   }
 
+  /**
+   * Test gets of simple blob with getChunkIdsOnly being true.
+   */
+  @Test
+  public void testSimpleBlobGetChunkIdsOnly() throws Exception {
+    blobSize = maxChunkSize;
+    doPut();
+    options =
+        new GetBlobOptionsInternal(new GetBlobOptionsBuilder().operationType(GetBlobOptions.OperationType.All).build(),
+            true, routerMetrics.ageAtGet);
+    getAndAssertSuccess();
+  }
```

Comment:
Thanks for adding this test. Is there a test case getChunkIdsOnly with composite blobs anywhere? If not, it may be worth adding

Reply:
added.